packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -8,6 +8,15 @@ import { applyMixins } from "../utilities/apply-mixins";
 import type { FoundationElementDefinition } from "../foundation-element";
 import { FormAssociatedButton } from "./button.form-associated";
 
+/**
+ * Define InstallTrigger as part of the Window to check for its presence in FireFox for working around the missing delegatesFocus support
+ */
+declare global {
+    interface Window {
+        InstallTrigger: any;
+    }
```

Comment:
In other places that we did something like this we've declared a new interface and use that in the tests, seems safer than touching the Window interface directly:    

`interface WindowWithInstallTrigger extends Window {
        InstallTrigger: any;
    }`

Reply:
Do you have an example with that? I guess I would then do something like 

`(window as WindowWithInstallTrigger).InstallTrigger`?